workers/ohsome_quality_analyst/api/response_models.py

```diff
@@ -35,6 +37,18 @@ def check_topic_dict(cls, value):
         return value
 
 
+class QualityDimensionMetadataResponse(ResponseBase):
+    result: dict[str, QualityDimension]
+
+    @validator("result")
+    @classmethod
+    def check_quality_dimension_dict(cls, value):
+        assert len(value) > 0
+        for key in value.keys():
+            QualityDimensionEnum(key)
+        return value
```

Comment:
I suggest using values to make clear that it is a dict:
```suggestion
    def check_quality_dimension_dict(cls, values):
        assert len(values) > 0
        for key in values.keys():
            QualityDimensionEnum(key)
        return values
```